docs/openapi.yaml

```diff
@@ -0,0 +1,73 @@
+openapi: 3.0.3
+info:
+  title: Alma SRU Proxy
+  version: '1.0'
+paths:
+  /:
+    get:
```

Comment:
Ser at consumes / produes er tatt vekk fra get, og jeg ser ikke produces definert ovenfor heller.
Det fungerer sikkert, men er det ikke greit å definere? (consumes er ikke relevant for get)